server/sonar-db-dao/src/main/java/org/sonar/db/organization/OrganizationDao.java

```diff
@@ -96,6 +97,17 @@ public void setDefaultTemplates(DbSession dbSession, String uuid, DefaultTemplat
     getMapper(dbSession).updateDefaultTemplates(uuid, defaultTemplates, now);
   }
 
+  public Optional<Integer> getDefaultGroupId(DbSession dbSession, String organizationUuid) {
```

Comment:
It's the same as done for default permission templates. But let's talk about this